gui/src/controls/virtualKeyboard.ts

```diff
@@ -112,6 +141,11 @@ module BABYLON.GUI {
                 }
 
                 this._connectedInputText.processKey(-1, key);
+                
+                if(this.shiftState === 1){

```

Comment:
Just wondering - shouldn't this be `> 1`? or is this a case that will never happen?

Reply:
Hi @RaananW 
That is only supposed to run when shiftState is 1, as in that case, only 1 letter will be capitalized and the shiftState reset to 0. 
When the shift key is clicked twice, shiftState = 2, all letters until shift is clicked again, are capitalized.